kafka/protocol/produce.py

```diff
@@ -78,6 +77,25 @@ class ProduceResponse_v5(Response):
     )
 
 
+class ProduceResponse_v6(Response):
+    """
+    Same as V5
```

Comment:
I'm sure it changed in some way, even if the change doesn't affect the structure of what we're using... so can you update the comment to mention what changed?

This applies to the other new Request/Response classes in this file as well. 

All you need is like 1-2 sentences of "Structurally the same as V5. But semantically X now indicates Y." or something like that...